src/components/product-card/ProductCard.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from "react";
+import { HeartIcon, RatingIcon, WishlistIcon } from "../../icons/icons";
+import { FilterContext, WishlistContext } from "../../context";
+import { CartContext } from "../../context/cart-context";
+import { Link } from "react-router-dom";
+export function ProductCard() {
+  const { getSortedArr } = useContext(FilterContext);
+  const { wishlistDataHandler, wishlistData } = useContext(WishlistContext);
+  const { dispatch, state } = useContext(CartContext);
+  return (
+    <div className="product-component">
+      <ul className="product-list">
+        {getSortedArr.map((item) => {
+          const { src, title, brand, offer_price, mrp, _id, rating } = item;
+          return (
+            <li key={_id}>
+              <div className="card card-vrt card-dismis">
+                <div className="card-img-container">
+                  <img className="card-img" src={src} alt={title} />
+                </div>
+                <div className="card-desc txt-left">
+                  <h3 className="card-desc-title">
+                    {title}
+                    <span className=" rating-num">
+                      {rating}
+                      <RatingIcon />
+                    </span>
+                  </h3>
+                  <p>by {brand}</p>
+                  <span className="card-price">Rs {offer_price}</span>
+                  <del className="card-mrp">Rs {mrp}</del>
```

Comment:
Give some space between these two prices. And if possible, increase the size of `offer_price` or make it bold. That way the offer price will stand out